### DISCUSSION TITLE
Bundle extension doesn't move CSS imports to the top

### DISCUSSION THREAD

**Discussion by GiovanniSalmeri** [datenstrom/community#733] [Report a bug] [answered]
I think that I stumbled upon a little bug in Bundle. `@import` declarations can be only at the beginning of a CSS file: when several CSSs are bundled, all `@import`s should therefore be moved at the top. Actually there is a `moveImportsToTop()` function, but it isn't applied to the final bundle. As a quick fix, I added after line 107

```
if ($type=="css") $fileDataBundle = CSS::moveImportsToTop($fileDataBundle);
```

and changed the function `moveImportsToTop` from `protected` to `public`. This seems to work, but I suspect there is a more elegant way to obtain this.

Steps for reproducing the bug: start with a clean installation; add `@import url("https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,400;0,700;1,400;1,700&display=swap");` at the top of stockholm.css and in the body declaration change font-family in `font-family: 'Merriweather', serif;`; now the site appears with the new font; install for example `Gallery`; now the site appears with the default serif font of the browser.

**Comment by schulle4u** [comment, marked as answer]
Thanks for the detailed bug report. An updated [Bundle extension](https://github.com/datenstrom/yellow-extensions/tree/master/source/bundle) is now available which should fix the problem.

**Reply by GiovanniSalmeri** [reply]
Thank you very much! Now it works perfectly.
